.github/workflows/compliance.yml

```diff
@@ -48,11 +50,19 @@ jobs:
         west update
         west zephyr-export
 
+    - name: Check if PR author is bot
+      if: contains(github.event.pull_request.user.login, 'dependabot[bot]')
+      run: |
+        echo "IS_BOT_PR=true" >> $GITHUB_ENV
+        echo "This PR is created by a bot and will skip GitLint and Identity modules"
+
     - name: Run Compliance Tests
```

Comment:
I would just disable this entirely if the author is a bot. What do you think @gmarull?

Reply:
Agree, makes sense

Reply:
changed